screens/MainScreen.js

```diff
@@ -0,0 +1,142 @@
+import React, {useEffect, useState} from 'react';
+import {KeyboardAwareScrollView} from 'react-native-keyboard-aware-scroll-view';
+import Icon from 'react-native-vector-icons/Ionicons';
+import MenuCard from '../Components/MenuCard';
+import {
+  Image,
+  ScrollView,
+  StyleSheet,
+  Text,
+  TextInput,
+  View,
+} from 'react-native';
+
+const MainScreen = props => {
+  const [data, setData] = useState([]);
+  const [searchTerm, setSearchTerm] = useState('');
+  const getFoodImages = async () => {
+    try {
+      const response = await fetch('https://burgers1.p.rapidapi.com/burgers', {
+        method: 'GET',
+        headers: {
+          'X-RapidAPI-Key':
+            'b13d4452e0msh022c443e270f56bp1d7113jsn7bdfb29892ae',
```

Comment:
Find out a way to store secrets properly so that we can keep it outside of code.